src/instruments/src/EFB/Dashboard/Widgets/WeatherWidget.tsx

```diff
@@ -243,13 +244,20 @@ const WeatherWidget = (props: WeatherWidgetProps) => {
                                         {metar.raw_text
                                             ? (
                                                 <>
-                                                    <ColoredMetar metar={metar} />
+                                                    {usingColordMetar
```

Comment:
```suggestion
                                                    {usingColoredMetar
```

Reply:
Typo is already in the variable otherwise it would not have worked :D - fixed and pushed